tests/cases/standalone/common/range/special_aggr.result

```diff
@@ -64,12 +64,12 @@ SELECT ts, host, first_value(addon ORDER BY val DESC) RANGE '5s', last_value(add
 | ts                  | host  | first_value(host.addon) ORDER BY [host.val DESC NULLS FIRST] RANGE 5s | last_value(host.addon) ORDER BY [host.val DESC NULLS FIRST] RANGE 5s |
 +---------------------+-------+-----------------------------------------------------------------------+----------------------------------------------------------------------+
 | 1970-01-01T00:00:00 | host1 | 3                                                                     | 1                                                                    |
-| 1970-01-01T00:00:05 | host1 | 4                                                                     | 4                                                                    |
+| 1970-01-01T00:00:05 | host1 | 4                                                                     | 6                                                                    |
```

Comment:
Is it expected?

Reply:
I think so. It's related to how "null"s are ordered in DataFusion, and it doesn't really matter here.